README.md

```
@@ -1,61 +1,137 @@
+├── **/*.spec.jsx   # Spec test files
+└── **/*.test.jsx   # Test files
+```
+
+
+## Running the code
 
-Ensure that you follow TDD approach to writing your code.
-The tests are available and marked as `skip`. Enable your tests by converting the
-`skip` into `test`
+1. Install dependencies:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)